### ISSUE TITLE
error:Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, Concat, ContrastiveLoss, Convolution, Crop, Data, 

### ISSUE THREAD

**Issue by striveallen** [tianzhi0549/CTPN#119]
I could successfully build, but I am not able to run the demo.When I execute the demo(no-gpu), the error is as follows:
Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, LRN, Log, Lstm, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Pooling, Power, ROIPooling, ReLU, Reduction, Reshape, Reverse, SPP, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, Transpose, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)
![微信截图_20200312104516](https://user-images.githubusercontent.com/61347112/76482294-b7acc000-644e-11ea-95ed-a074ba554530.png)

**Comment by tianzhi0549** [comment]
@striveallen Please make sure you compile caffe with Python support.

**Comment by striveallen** [comment]
Thank you! @tianzhi0549
